### DISCUSSION TITLE
Can you install atlas on surface pro?

### DISCUSSION THREAD

**Discussion by Tw3ak** [Atlas-OS/Atlas#1271] [Help & Support] [answered]
Hello i am interested in installing atlasos on my wifes surface pro. I have atlas installed on 2 other laptops and my desktop, I told my wife it should be no problem as her surface is windows 11 and it has worked fine on any other laptop i have installed it on, But then she asked me if her stylus and everything would still work? And i forgot about that lol i said ohh i don't know about that as the surface can also be used a tablet and touch screen with stylus.

So i told her i would research it first and i can't find any posts on google,redit, or here on if it can be done and still have full functionality as before with stylus and other functions that make it more then just a laptop. So i decided to make a post as see if anyone knows if it would work fine or if it will break functions of her surface pro.

Thanks for any help on this.

**Comment by imcharliesparks** [comment]
@Tw3ak did you ever hear back about this? I'm wondering if AtlasOS supports all of the pen and touchscreen features or if those are trimmed out, as well.

**Comment by bickbick** [comment]
I can confirm Atlas works fine on Surface Go MK1, the original low spec Surface. Camera, bluetooth, sound, touchscreen, all ok. It took about an hour for the script to run, looked like it would never finish! But it did and works great.

**Comment by Aldighieri22** [comment, marked as answer]
I instaled on a Surface Pro 7+ using Windows 11 Home, and its worked perfect!

**Comment by ClemKey** [comment]
Hello,
Installed on Surface PRO 2 - Windows 11 Pro 24h2 + Atlas os 0.4.1
Works perfectly too

**Comment by Tw3ak** [comment]
Great thanks for responses guys i'll try it on mine :)
